UKHO.ChromeDriver.BinarySync/Functions/Get-ChromeVersion.ps1

```diff
@@ -2,16 +2,26 @@ function Get-ChromeVersion {
     [CmdletBinding()]
     [OutputType([System.Diagnostics.FileVersionInfo])]
     param (
+        [Parameter()]
+        [switch]
+        $IncludeBeta
     )
 
     begin {
+        $chromePaths = @(
+            "C:\Program Files\Google\Chrome\Application\chrome.exe",
+            "C:\Program Files (x86)\Google\Chrome\Application\chrome.exe"
+        )
+        if ($IncludeBeta) {
+            $chromePaths = @(
+                "C:\Program Files\Google\Chrome Beta\Application\chrome.exe"
+                "C:\Program Files (x86)\Google\Chrome Beta\Application\chrome.exe"
+            ) + $chromePaths
```

Comment:
Sorry for the dumb question... so if its Beta then it will get the 'non' Beta chromePaths and the Beta paths?

Reply:
Yes I was playing around with this and I can't find a nice way of configuring this. It would be a pain to have to choose either/or but I wanted the installation of the beta to be opt-in.